pandas/core/frame.py

```diff
@@ -598,6 +598,12 @@ def __init__(
         if data is None:
             data = {}
         if dtype is not None:
+            if isinstance(dtype, str):
+                if dtype == "Int64":
+                    dtype = "int64"
+                elif dtype == "Float64":
+                    dtype = "float64"
```

Comment:
this is going to change behavior in an unwanted way.  The issue is with 'fillna' with an Int64 dtype, not the existence of Int64 dtypes

Reply:
Agreed - as mentioned above in the description this contribution is for a final class project. Just trying to get a feel for contributing to open source software.

Reply:
Are you planning to follow-up on this?  That's an important part of the contribution process!